easybuild/easyconfigs/c/CMake/CMake-3.8.2-GCCcore-6.4.0.eb

```diff
@@ -0,0 +1,38 @@
+easyblock = 'ConfigureMake'
+
+name = 'CMake'
+version = '3.8.2'
+
+homepage = 'http://www.cmake.org'
+
+description = """
+ CMake, the cross-platform, open-source build system.  CMake is a family of
+ tools designed to build, test and package software.
+"""
+
+toolchain = {'name': 'GCCcore', 'version': '6.4.0'}
+
+source_urls = ['http://www.cmake.org/files/v%(version_major_minor)s']
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['da3072794eb4c09f2d782fcee043847b99bb4cf8d4573978d9b2024214d6e92d']
+
+configopts = '-- -DCMAKE_USE_OPENSSL=1 -DCMAKE_PREFIX_PATH=$EBROOTNCURSES'
+
+builddependencies = [
+    ('binutils', '2.28'),
+]
+
+dependencies = [
+    ('ncurses', '6.0'),
+]
```

Comment:
@JackPerdue Why did you drop the commented out dependency on `OpenSSL` here?

I think it makes sense to retain that, for people who prefer providing OpenSSL through EasyBuild?